corehq/apps/accounting/invoicing.py

```diff
@@ -677,7 +677,9 @@ def num_users(self):
 
     @property
     def unit_description(self):
-        if self.num_excess_users > 0:
+        quarterly_invoice = self.invoice.is_customer_invoice and \
+                            self.invoice.account.invoicing_plan != InvoicingPlan.MONTHLY
```

Comment:
So we are assuming that if its not a monthly plan, its quarterly?
if thats true, that sounds brittle to me. Like if a new plan is added, say annual. This would be true.